tasks/main.yml

```diff
@@ -292,7 +278,133 @@
 
-    - name: Remove the {{ __mssql_sql_tempfile.path }} file
+    - name: Remove the tempfile
       file:
         path: "{{ __mssql_sql_tempfile.path }}"
         state: absent
+
+- name: Configure the sqlagent setting
+  include_tasks: mssql_conf_setting.yml
+  vars:
+    __mssql_conf_setting: "sqlagent enabled"
+    __mssql_conf_setting_value: "{{ mssql_enable_sql_agent }}"
+  when: mssql_enable_sql_agent is not none
+
+- name: Ensure the {{ __mssql_server_fts_packages }} package
+  package:
+    name: "{{ __mssql_server_fts_packages }}"
+    state: "{{ 'present' if mssql_install_fts | bool else 'absent' }}"
+  when: mssql_install_fts is not none
+  notify: Restart the mssql-server service
+
+- name: Configure HA
+  when: mssql_enable_ha is not none
+  block:
+    - name: Ensure the {{ __mssql_server_ha_packages }} package
+      package:
+        name: "{{ __mssql_server_ha_packages }}"
+        state: "{{ 'present' if mssql_enable_ha | bool else 'absent' }}"
+      notify: Restart the mssql-server service
+
+    - name: Configure the hadrenabled setting
+      include_tasks: mssql_conf_setting.yml
+      vars:
+        __mssql_conf_setting: "hadr hadrenabled"
+        __mssql_conf_setting_value: "{{ mssql_enable_ha }}"
+
+- name: Tune MSSQL for FUA-capable storage
+  when: mssql_tune_for_fua_storage is not none
+  block:
+    - name: Check if the 3979 trace flag is enabled
+      command: /opt/mssql/bin/mssql-conf get traceflag
+      ignore_errors: true
+      changed_when: false
+      register: __mssql_get_traceflag
+
+    - name: Set the 3979 traceflag
+      command: >-
+        /opt/mssql/bin/mssql-conf traceflag 3979 "on"
+      when:
+        - mssql_tune_for_fua_storage | bool
+        - "'3979' not in __mssql_get_traceflag.stdout"
+
+    - name: Unset the 3979 traceflag
+      command: >-
+        /opt/mssql/bin/mssql-conf traceflag 3979 "off"
+      when:
+        - not mssql_tune_for_fua_storage | bool
+        - "'3979' in __mssql_get_traceflag.stdout"
+
+    - name: Configure the alternatewritethrough setting
+      include_tasks: mssql_conf_setting.yml
+      vars:
+        __mssql_conf_setting: "control alternatewritethrough"
+        __mssql_conf_setting_value: "false"
+
+    - name: Configure the writethrough setting
+      include_tasks: mssql_conf_setting.yml
+      vars:
+        __mssql_conf_setting: "control writethrough"
+        __mssql_conf_setting_value: "{{ mssql_tune_for_fua_storage }}"
+
+- name: Configure MSSQL to encrypt connections
+  when: mssql_tls.enable | bool
+  block:
+    - name: Copy cert and private key to the host
+      include_tasks: copy_certs.yml
+      with_items:
+        - "{{ mssql_tls.cert }}"
+        - "{{ mssql_tls.private_key }}"
+
+    - name: Configure the tlscert setting
+      include_tasks: mssql_conf_setting.yml
+      vars:
+        __mssql_conf_setting: "network tlscert"
+        __mssql_conf_setting_value: >-
+          /etc/pki/tls/certs/{{ mssql_tls.cert }}
```

Comment:
`           /etc/pki/tls/certs/{{ mssql_tls.cert | basename }}`

Reply:
Thanks a lot for this fix, I implemented it.